code/modules/mob/living/simple_animal/abnormality/he/pinocchio.dm

```diff
@@ -0,0 +1,274 @@
+//Coded by Coxswain
+/mob/living/simple_animal/hostile/abnormality/pinocchio
+	name = "Pinocchio"
+	desc = "A wooden humanoid puppet, it hums to itself with childlike delight."
+	icon = 'ModularTegustation/Teguicons/32x32.dmi'
+	icon_state = "pinocchio"
+	icon_living = "pinocchio"
+	maxHealth = 600
+	health = 600
+	threat_level = HE_LEVEL
+	can_breach = TRUE
+	start_qliphoth = 1
+	speak_emote = list("creaks", "snaps")
+	work_chances = list(
+		ABNORMALITY_WORK_INSTINCT = 45,
+		ABNORMALITY_WORK_INSIGHT = 45,
+		ABNORMALITY_WORK_ATTACHMENT = 45,
+		ABNORMALITY_WORK_REPRESSION = 45,
+		"Lying is Bad!" = 0
+	)
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 1.2, WHITE_DAMAGE = 0.5, BLACK_DAMAGE = 0.7, PALE_DAMAGE = 0.9)
+	work_damage_amount = 8
+	work_damage_type = WHITE_DAMAGE
+	max_boxes = 16
+
+	ego_list = list(
+			/datum/ego_datum/armor/marionette,
+			/datum/ego_datum/weapon/marionette
+		)
+	gift_type = /datum/ego_gifts/marionette
+	abnormality_origin = ABNORMALITY_ORIGIN_RUINA
+
+	var/lying = FALSE
+	var/caught_lie = FALSE
+	var/mob/living/carbon/human/species/pinocchio/realboy = null
+	var/list/modular_work_chance = list(
```

Comment:
This could probably be replaced with a randomization whenever the console is opened, at a chance.